composer.json

```diff
@@ -22,7 +22,9 @@
     "zetacomponents/mail": "dev-1.7-civi",
     "phpoffice/phpword": "^0.13.0",
     "pear/Validate_Finance_CreditCard": "dev-master",
-    "civicrm/civicrm-cxn-rpc": "~0.16.12.05"
+    "civicrm/civicrm-cxn-rpc": "~0.16.12.05",
+    "pear/Net_SMTP": "1.6.*",
```

Comment:
It might be worth noting that composer [recommends using the caret](https://getcomposer.org/doc/articles/versions.md#caret) themselves, but this would mean all versions under 2.0 would be allowed.